package.json

```diff
@@ -4,13 +4,18 @@
   "description": "Standalone WWW client for Scratch",
   "scripts": {
     "start": "node ./dev-server/index.js",
-    "test": "npm run test:lint && npm run build && npm run test:tap",
+    "test": "npm run test:lint && npm run build && npm run test:unit:tap",
```

Comment:
This should be `run test:unit`, right? So it'll run jest tests too?

Reply:
Yes.  I changed it briefly for an interstitial commit and forgot to change it back.  I'll add that back in in a moment.